components/RegistrationForm.tsx

```diff
@@ -302,116 +204,79 @@ const RegistrationForm = ({
-        setSubmitLoading(false);
-      }
-    },
-    [mutation.isSuccess, submitLoading, mutation.isLoading] 
-  );
-
-  let render;
-
-  if (isRegistrationOpen) {
-    render = (
-      <>
-        <Typography gutterBottom>
-          Please review the&nbsp;
-          <a href="#contest-outline">
-            Contest Outline
-          </a>
-          &nbsp;with your parent or guardian before registering to ensure you meet the eligibility requirements and understand your commitment to Digital Waves.
-        </Typography>
-        <Typography>
-          You may also register for our event on&nbsp;
-          <a
-            href="https://www.eventbrite.ca/e/digital-waves-participant-registration-tickets-189810657787"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Eventbrite
-          </a>
-          .
-        </Typography>
-        {
-          submitSuccess
-            ? (
-              <>
-                {
-                  duplicateParentWarning
-                    ? (
-                      <Typography className={classes.typographyDuplicateParent}>
-                        <strong>Heads up!</strong>
-                        <br /><br />
-                        The parent/guardian email associated with this registration is already subscribed to Digital Waves updates.
-                        To reduce clutter in their inbox, they will not receieve the registration information again.
-                        <br /><br />
-                        If you feel like this is in error, please contact&nbsp;
-                        <a href="mailto:info@digitalwavesnl.ca">
-                          info@digitalwavesnl.ca
-                        </a>
-                        &nbsp;and we will be happy to assist you.
-                      </Typography>
-                    )
-                    : null
-                }
-                <Typography className={classes.form}>
-                  <strong>
-                    Thanks for registering to participate in Digital Waves!
-                    You should receive an email from us within the next 24 hours.
-                  </strong>
-                </Typography>
-                <Typography>
-                  <strong>
-                    If you don&rsquo;t see anything in your inbox after this time,
-                    check your spam folder or contact&nbsp;
-                    <a href="mailto:info@digitalwavesnl.ca">
-                      info@digitalwavesnl.ca
-                    </a>
-                    &nbsp;to verify your registration.
-                  </strong>
-                </Typography>
-              </>
-            )
-            : (
+  return (
+    <div className={classes.root}>
+      {
+        submitSuccess
+          ? (
+            <>
+              <Typography
+                variant="h2"
+                className={classes.typographyH2}
+              >
+                Congrats, you&rsquo;re all set!
```

Comment:
The hash goes to an ID located in `Program2023Register.tsx` component. It likely didn't work for you because you were testing the form multiple times and the hash was already in the browser. I can add a quick fix for this other case next chance I look, not non-blocking issue IMO.